### ISSUE TITLE
Cannot commit to branch: Clippy warnings about 'unnecessary hashes around raw string literal'

### ISSUE THREAD

**Issue by sluedecke** [mtkennerly/ludusavi#262]
### Ludusavi version

v0.21.0

### Operating system

Windows

### Installation method

Cargo

### Description

Trying to run `cargo clippy` shows some warnings:

using rustc 1.72.0

``` shell
$ cargo clippy
warning: unnecessary hashes around raw string literal
   --> src/lang.rs:209:63
    |
209 | ...(|| Regex::new(r#"([^\r\n ]) {2,}"#).unwrap());
    |                   ^^^^^^^^^^^^^^^^^^^^ help: try: `r"([^\r\n ]) {2,}"`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_raw_string_hashes
    = note: `#[warn(clippy::needless_raw_string_hashes)]` on by default

warning: unnecessary hashes around raw string literal
   --> src/lang.rs:210:62
    |
210 | ...new(r#"([^\r\n ])[\r\n]([^\r\n ])"#).unwrap());
    |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try: `r"([^\r\n ])[\r\n]([^\r\n ])"`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_raw_string_hashes

warning: unnecessary hashes around raw string literal
   --> src/lang.rs:211:67
    |
211 | ...:new(|| Regex::new(r#"([^\r\n ])[\r\n]{2,}([^\r\n ])"#).unwrap());
    |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try: `r"([^\r\n ])[\r\n]{2,}([^\r\n ])"`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_raw_string_hashes

warning: unnecessary hashes around raw string literal
  --> src/scan/title.rs:18:62
   |
18 | static RE_YEAR_SUFFIX: Lazy<Regex> = Lazy::new(|| Regex::new(r#" \(\d+\)$"#).unwrap());
   |                                                              ^^^^^^^^^^^^^^ help: try: `r" \(\d+\)$"`
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_raw_string_hashes

warning: `ludusavi` (bin "ludusavi") generated 4 warnings (run `cargo clippy --fix --bin "ludusavi"` to apply 4 suggestions)
    Finished dev [optimized + debuginfo] target(s) in 0.13s
```

**Comment by sluedecke** [comment]
Unfortunately the warnings become errors when I try to commit to my branch https://github.com/sluedecke/ludusavi/tree/feature/launcher after I merged with master from upstream.

Probably my fault, but I don't see the solution ...

**Comment by mtkennerly** [comment]
> Probably my fault, but I don't see the solution ...

Nah, it's not your fault. It's just a new lint with Rust 1.72, and I hadn't updated master yet. The workflow always uses the latest version of Rust. Fixed now in a3b0355c6bd05058c7afcd087d84ba60427c3434.
